### PR TITLE
Property widget: Fix selection handling

### PR DESCRIPTION
**Pull request by saskliutas**
Closes https://github.com/iTwin/viewer-components-react/issues/611

### CODE REVIEW

**Review by grigasp — APPROVED** [review]
LGTM. I believe you may need to regenerate e2e test screenshots after merging with master.

File: packages/itwin/property-grid/playwright.config.ts [hunk 1]
```diff
@@ -75,6 +75,13 @@ export default defineConfig({
     // },
   ],
 
+  expect: {
+    toHaveScreenshot: {
+      /* Allow some difference between screenshots as rendering might differ on different machines. */
+      maxDiffPixelRatio: 0.01
```
Comment (grigasp) [inline packages/itwin/property-grid/playwright.config.ts:81]: why do we need this? 😞

File: packages/itwin/property-grid/src/e2e-tests/PropertyGrid.test.ts-snapshots/should-match-image-snapshot-stage-panel-size---narrow-multiple-elements-selected-1-chromium-win32.png
Comment (grigasp) [inline packages/itwin/property-grid/src/e2e-tests/PropertyGrid.test.ts-snapshots/should-match-image-snapshot-stage-panel-size---narrow-multiple-elements-selected-1-chromium-win32.png:1]: looks like property labels and values are in different fonts - mind having a look at this?

Comment (saskliutas) [reply packages/itwin/property-grid/src/e2e-tests/PropertyGrid.test.ts-snapshots/should-match-image-snapshot-stage-panel-size---narrow-multiple-elements-selected-1-chromium-win32.png:1]: They might be in different font or maybe just different size. I tried to find font similar to the one @medikornov used to created initial snapshots. 

From what I found out it looks like that different machines might render same page differently https://github.com/microsoft/playwright/issues/20097#issuecomment-1419220152
Probably the best solution would be to have a docker container and run e2e test in it.

File: packages/itwin/property-grid/playwright.config.ts (hunk 1 above)
Comment (saskliutas) [reply packages/itwin/property-grid/playwright.config.ts:81]: This should answer https://github.com/iTwin/viewer-components-react/pull/613/files/f2c46feeef0e972e253035a5b75ae86b44cf8993..460b500656c196b72374d2e2ca83fa59ee6b85c0#r1288295845

File: packages/itwin/property-grid/src/e2e-tests/PropertyGrid.test.ts-snapshots/should-match-image-snapshot-stage-panel-size---narrow-multiple-elements-selected-1-chromium-win32.png
Comment (grigasp) [reply packages/itwin/property-grid/src/e2e-tests/PropertyGrid.test.ts-snapshots/should-match-image-snapshot-stage-panel-size---narrow-multiple-elements-selected-1-chromium-win32.png:1]: That's sad. I filed https://github.com/iTwin/viewer-components-react/issues/614 to handle the situation, in the meantime let's keep the tolerance threshold.

**Pull request merged**